integrationTests/package.json

```diff
@@ -8,9 +8,10 @@
     "depcheck": "depcheck",
     "lint": "eslint . --ext .ts,.js --quiet --fix",
     "pretest": "npm run lint && npm run depcheck",
-    "test:ts": "nyc mocha \"tests/*.test.ts\"",
-    "test:js": "nyc mocha \"tests/*.test.js\"",
-    "test": "npm run test:ts && npm run test:js",
+    "test:basic:ts": "nyc mocha \"tests/basic/*.test.ts\"",
```

Comment:
I do not love this. Basically it forces following a strict standard where we just don't run a set of tests by default. 

I would prefer it if you don't have redis set up for the redis test that the tests just fail by default.  Here when you run the tests it just skips the redis test by default.

Reply:
We have initially decided to run redis tests separately. 
also test:all does not ignore any tests. Do u want to change this still?

Reply:
I really don't like this because if you add something that is not in "basic" it doesn't get run at all and you might not realize the test isn't running,.

Reply:
I c what u r saying, 
let me include all tests in test script and will let the tests fail when redis is not running..

Reply:
Changed, test now will test all files under tests folder other than javascript redis test files